token/token.go

```diff
@@ -600,20 +601,26 @@ func ToNumber(value string) *NumberValue {
 	}
 }
 
-func looksLikeTimeValue(value string) bool {
-	for i, c := range value {
-		switch c {
-		case ':', '1', '2', '3', '4', '5', '6', '7', '8', '9':
-			continue
-		case '0':
-			if i == 0 {
-				return false
-			}
-			continue
+// This is a subset of the formats permitted by the regular expression
+// defined at http://yaml.org/type/timestamp.html. Note that time.Parse
+// cannot handle: "2001-12-14 21:59:43.10 -5" from the examples.
```

Comment:
Why didn't you provide a layout to handle this case ?

Reply:
Yes I've actually tried supporting the layout, but Go does not seem to support the "-timezone" representation...

https://go.dev/play/p/a5UvI5cUlJy

Reply:
OK ! I understand !